README.md

```diff
@@ -14,7 +14,7 @@ Documentation for Interchange, including examples, a brief user guide, release h
 
 ## How to cite
 
-Please cite Interchange using the [Zenodo record](https://zenodo.org/record/8147765) of the [latest release](https://zenodo.org/record/8147765) or the version that was used. The BibTeX reference of the latest release can be found [here](https://zenodo.org/record/8147765/export/hx).
+Please cite Interchange using the [Zenodo record](https://doi.org/10.5281/zenodo.10068101) of the [latest release](https://zenodo.org/doi/10.5281/zenodo.8147764) or the version that was used. The BibTeX reference of the latest release can be found [here](https://zenodo.org/record/8147765/export/hx).
```

Comment:
```suggestion
Please cite Interchange using the [Zenodo record](https://doi.org/10.5281/zenodo.10068101) of the [latest release](https://zenodo.org/doi/10.5281/zenodo.8147764) or the version that was used. The BibTeX reference of the latest release can be found [here](https://zenodo.org/record/8147764/export/hx).
```

Reply:
Ah - so the `10068101` URL wouldn't need to be updated, but I think the `814....` would?

Reply:
That's right, one of them is evergreen -- it's the doi for the entire record, and always resolves to the latest version. 

Maybe the right answer is to ONLY include that link in the README, but the way it was written it had two links so I didn't want to change that without knowing more about why it was written that way.